internal/installcfg/installcfg.go

```diff
@@ -340,6 +361,11 @@ func getInstallConfig(log logrus.FieldLogger, cluster *common.Cluster, addRhCa b
 	if err != nil {
 		return nil, err
 	}
+	// CA for mirror registries and RH CA are mutually exclusive
+	if cluster.ImageInfo.CaConfig != "" {
+		addRhCa = true
```

Comment:
This will now add the Red Hat CA any time we have some other CA in the config? Why? What does one have to do with the other?

Reply:
No, this will only add a a user-provided CA, see line 367.  RedHat CA will be added only in case user-provided CA does not exists and red hat CA was provided

Reply:
I think re-using `addRhCa` makes this very difficult to understand.
Maybe use `addRhCa || cluster.ImageInfo.CaConfig != ""` as the condition on 369?

I think making this easier to read is worth evaluating `cluster.ImageInfo.CaConfig != ""` multiple times.

Reply:
@yevgeny-shnaidman does this make sense? I don't like using `addRhCa` if the CA we're adding isn't the Red Hat one.